app/com/arpnetworking/metrics/portal/alerts/impl/PluggableAlertRepository.java

```diff
@@ -120,6 +121,28 @@ public PluggableAlertRepository(
         _organization = new DefaultOrganization.Builder().setId(org).build();
     }
 
+    /**
+     * Test constructor.
+     *
+     * Instantiates a repo with a fixed set of alerts that will never be
+     * updated.
+     *
+     * @param objectMapper The object mapper to use for alert deserialization.
+     * @param org The organization to group the alerts under.
+     * @param alerts The set of alerts to place into this repository.
+     */
+    @VisibleForTesting
+    public PluggableAlertRepository(
```

Comment:
I opted for exposing this constructor because
* PluggableAlertRepository is already an in-memory repository, albeit unmodifiable. Since the controller doesn't have any mutation endpoints right now we don't need those anyway.
* Alternative would be: Create an explicit `MapAlertRepository` that is just a dumber pluggable alert repository. This seemed unnecessary and we can add it once we need mutator methods on the conroller.

Reply:
Isn't the answer just to create a `MapConfigProvider`? This would take a map, serialize it, and then provide it to the repository. Seems like you did that pattern already with `NullConfigProvider`.

Reply:
I didn't do this originally because the serialization code is all within an inner class of PluggableAlertRepository, and it would have to be exposed in order to be reused in this test class. Let me see about doing this.